new-log-viewer/src/services/LogFileManager/index.ts

```diff
@@ -168,33 +186,33 @@ class LogFileManager {
      * numbers, and the line number of the first line in the cursor identified event.
      * @throws {Error} if any error occurs during decode.
      */
-    loadPage (cursor: CursorType): {
-        beginLineNumToLogEventNum: BeginLineNumToLogEventNumMap,
-        cursorLineNum: number
-        logEventNum: number
-        logs: string,
-        pageNum: number
-    } {
+    loadPage (cursor: CursorType):
+        WorkerResp<WORKER_RESP_CODE.PAGE_DATA> {
         console.debug(`loadPage: cursor=${JSON.stringify(cursor)}`);
+        const filteredLogEventMap = this.#decoder.getFilteredLogEventMap();
+        const activeNumEvents: number = filteredLogEventMap ?
```

Comment:
How about `numFilteredEvents`? When no filter is active, I think it's intuitive that `numFilteredEvents == #numEvents`.

Reply:
How about `numSelectedEvents`? I just feel like `activeNumEvents` doesn't convey much meaning.

Reply:
How about `numActiveEvents`?